### DISCUSSION TITLE
German translation (Wünsche / Änderungen)

### DISCUSSION THREAD

**Discussion by hliebscher** [OrcaSlicer/OrcaSlicer#1058] [Q&A]
I want to collect all wishes for changes here and implement them as best as possible for the community.

Natürlich gern auf deutsch.

Manchmal gibt es nur einen String im OrcaSlicer für mehrere Bedeutungen. Und da bestimmte Drucker andere Menüs haben
und die Betriebsysteme andere Layouts, ist es schwierig es immer richtig zu übersetzen.
Und inzwischen sind es 12.000 Zeilen mit reichlich Platz für Fehler ;-)

**Comment by hliebscher** [comment]
1. Frage ist der Dialog in Ordnung ?

oder soll da nur 
Drucken
Alle Drucken
Senden
Alle Senden

rein?



https://user-images.githubusercontent.com/9274162/238174732-76fcef64-6fda-4f42-8b07-fdd536e82aba.jpg

**Comment by ClamsyHub** [comment]
Hi Heiko,
also ich finde das Menü gemäß dem Screenshot in Ordnung. Oder würdest Du die anderen Punkte separat zur Auswahl stellen? Es ist halt immer "schlecht" wenn man gewohnte Funktionen wo anders "versteckt" oder sogar entfernt. Es gibt sicherlich den ein oder anderen, der die Punkte nutzt und dann schimpft wenn diese nicht mehr zur Auswahl stünden.

**Reply by hliebscher** [reply]
ja leider wird für den X1C im Untermenü "send" verwendet und der Begriff war mir zu kurz und ist alle Platten senden rausgekommen
und das steht jetzt im anderen Menü aber ich ändere das...

**Comment by Hamsterberg** [comment]
![Bildschirmfoto zu 2025-03-23 11-05-27](https://github.com/user-attachments/assets/89aa2ace-1ac3-402b-a500-605d4bdce3a5)

Kann man das "Speere" in "Sperre" ändern? Ist schon seit einigen Versionen fehlerhaft.
